spec/support/raven.rb

```diff
@@ -0,0 +1,12 @@
+# frozen_string_literal: true
+
+require "raven"
+require "raven/transports/dummy"
+
+Raven::Context.prepend(Lamian::RavenContextExtension)
+
+Raven.configure do |config|
+  config.dsn = "dummy://public@example.com/project-id"
+  config.encoding = "json"
+  config.logger = Logger.new(nil)
+end
```

Comment:
You c an inline it inside spec_helper.

If you don't want, please extract "cool logers" context from there and use `Dir[File.join(_dir_, "support/**/*.rb")].each { |x| require(x) }` inside spec_helper